src/CMakeLists.txt

```diff
@@ -26,10 +26,6 @@ else()
   set (HAS_LOMIRI_ACCTSERVICE_SYSTEMSOUND_SETTINGS OFF)
 endif()
 
-if(EXISTS "/usr/share/accountsservice/interfaces/com.lomiri.AccountsService.Sound.xml")
-  set (VALA_DEFINE_ACCTSERVICE_SOUND_SETTINGS "--define=HAS_LOMIRI_ACCTSERVICE_SOUND_SETTINGS")
-endif()
-
```

Comment:
Please don't attach the presence of the org.lomiri.sound.gschema.xml file and guess from that the AccountsService file com.lomiri.AccountsService.Sound.xml must also exist.

IMHO, we should differentiate between HAS_LOMIRI_SOUND_SCHEMA and HAS_LOMIRI_ACCTSERVICE_SOUND_SETTINGS.

With this PR you weaken this differentiation.

Reply:
Ignoring, as we agreed to simplify this in the next PR.